### ISSUE TITLE
Unique Validations prevent a Model from being updated.

### ISSUE THREAD

**Issue by dalberto** [maritz/nohm#82]
I tried manually updating a model by changing a single property.

i.e.

``` javascript
model.id = 'some_id';
model.property('some_property', 'some_value');

model.save( function (err) {
  if (err) {
     ...
  }
    ...
});
```

The property along with a few others have a uniqueness constraint.

Saving the model fails with an error message of "invalid". I removed the uniqueness constraint and the model can be saved and updated. It would be great to be able to update a model under this corner case.

**Comment by maritz** [comment]
Using Symbols to make the used id properly private will allow us to safely make decisions about this problem.

**Comment by sam2x** [comment]
Running the provided sample with skip_validation_and_unique_indexes set to true works : 

     moddedJetta.save({skip_validation_and_unique_indexes: true}, function(err))

When you dev this project, wasnt this parameter supposed to deal with such case ?

**Comment by maritz** [comment]
No, the original use-case for that option can be found [here](https://github.com/maritz/nohm/pull/84).

This is an entirely different problem.

**Comment by sam2x** [comment]
Ok will check that. Btw I misscliked on github page when i wanted to reset my forked repo, I think I "removed" my own rights on this repo -_-, can you give them back ? thx.

**Comment by maritz** [comment]
This fix is now available in v2.0.0-alpha.7, however I do not plan on backporting it to 0.9.x.

If anyone else runs into this issue and is stuck on 0.9.x or 1.x, please make sure to always load models before saving them.
